tasks/main.yml

```diff
@@ -215,16 +264,48 @@
       name: consul
       state: started
       enabled: yes
+    when: ansible_os_family != "Windows"
+
+  - name: Install NSSM
+    win_chocolatey:
+      name: nssm
+      state: latest
+
+  - name: Start Consul on Windows
+    win_nssm:
```

Comment:
NSSM is used to wrap consul.exe into a Windows-friendly service. Without it we can't use the regular `win_service` module because services don't let you set configuration variables.